src/devices/tuya.ts

```diff
@@ -737,6 +737,28 @@ const definitions: Definition[] = [
         toZigbee: [],
         exposes: [e.carbon_monoxide(), e.co()],
     },
+    {
+        fingerprint: tuya.fingerprint('TS0601', ['_TZE200_rjxqso4a']),
```

Comment:
Please move it to `moes.js` and remove the `whitelabel` (update `model`/`vendor` instead)

Reply:
ok